integrationTests/consensus/consensus_test.go

```diff
@@ -174,7 +174,7 @@ func TestConsensusBLSFullTest(t *testing.T) {
 	runFullConsensusTest(t, blsConsensusType)
 }
 
-func runConsensusWithNotEnoughValidators(t *testing.T, consensusType string) {
+func runConsensusWithNotEnoughValidators(t *testing.T, _ string) {
```

Comment:
if consensusType is not used, then both of the tests are ran through BLS.

Reply:
good catch. will fix

Reply:
done